spec/system/admin_company_value_spec.rb

```diff
@@ -0,0 +1,104 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Modifying company values', type: :system do
+  let(:admin_user) { create(:admin_user) }
+  let!(:company_value) { create(:company_value) }
+
+  before do
+    driven_by(:rack_test)
+    sign_in admin_user
+  end
+
+  context 'when creating a new company value' do
+    it 'enables to create a new company value' do
+      expect(CompanyValue.count).to eq 1
+
+      go_to_form_new_company_value
```

Comment:
using custom methods in specs is not a common practice - do not treat repetition here as something bad - it will be easier to read, also if you really don't want to repeat, you can simply use `before` block